smart-contracts/test/helpers/deployLocks.js

```diff
@@ -38,7 +38,7 @@ module.exports = function deployLocks(
       } else {
         const tx = await createCall
         // THIS API IS LIKELY TO BREAK BECAUSE IT ASSUMES SO MUCH
-        const evt = tx.logs[1]
+        const evt = tx.logs.find(v => v.event === 'NewLock')
```

Comment:
I'm not familiar with this syntax...it looks like a way to filter logs by event name?

Reply:
Yea, it's JS shorthand to write a loop looking for a certain element.  for each element (`v` here) run the function until one returns true

Reply:
cool, thx